apps/web/src/utils/wagmi.ts

```diff
@@ -1,134 +1,57 @@
-    // Error: To use QR modal, please install @walletconnect/modal package
-    showQrModal: process.env.NODE_ENV !== 'test',
-    projectId: 'e542ff314e26ff34de2d4fba98db70bb',
-  },
+export const walletConnectNoQrCodeConnector = walletConnect({
+  projectId: 'e542ff314e26ff34de2d4fba98db70bb',
+  showQrModal: false,
 })
 
-export const walletConnectNoQrCodeConnector = new WalletConnectConnector({
-  chains,
-  options: {
-    showQrModal: false,
-    projectId: 'e542ff314e26ff34de2d4fba98db70bb',
-  },
-})
-
-export const metaMaskConnector = new MetaMaskConnector({
-  chains,
-  options: {
-    shimDisconnect: false,
-  },
-})
-
-const bloctoConnector = new BloctoConnector({
-  chains,
-  options: {
-    defaultChainId: 56,
-    appId: 'e2f2f0cd-3ceb-4dec-b293-bb555f2ed5af',
-  },
-})
-
-const ledgerConnector = new LedgerConnector({
-  chains,
-  options: {
-    projectId: 'e542ff314e26ff34de2d4fba98db70bb',
-  },
+const bloctoConnector = blocto({
+  appId: 'e2f2f0cd-3ceb-4dec-b293-bb555f2ed5af',
 })
 
-export const bscConnector = new BinanceWalletConnector({ chains })
-
-export const trustWalletConnector = new TrustWalletConnector({
-  chains,
-  options: {
-    shimDisconnect: false,
-    shimChainChangedDisconnect: true,
-  },
-})
-
-export const cyberWalletConnector = isCyberWallet()
-  ? new CyberWalletConnector({
-      chains: chains as any,
-      options: {
-        name: 'PancakeSwap',
-        appId: 'b825cd87-2db3-456d-b108-d61e74d89771',
-      },
-    })
-  : undefined
-
-const BinanceW3WConnector = getWagmiConnector()
-export const binanceWeb3WalletConnector = new BinanceW3WConnector({
-  chains,
-  options: {
-    chainId: ChainId.BSC,
-  },
-})
+export const CONNECTORS_INIT_LENGTH = 5
 
 export const noopStorage = {
   getItem: (_key: any) => '',
   setItem: (_key: any, _value: any) => null,
   removeItem: (_key: any) => null,
 }
-
 export const wagmiConfig = createConfig({
-  storage: createStorage({
-    storage: typeof window !== 'undefined' ? window.localStorage : noopStorage,
-    key: 'wagmi_v1.1',
-  }),
-  autoConnect: false,
-  publicClient,
+  chains: [mainnet, bsc],
+  client({ chain }) {
+    return createClient({ chain, transport: http() })
+  },
   connectors: [
-    metaMaskConnector,
-    injectedConnector,
-    coinbaseConnector,
+    metaMask(),
     walletConnectConnector,
-    bscConnector,
-    // @ts-ignore FIXME: wagmi
+    walletConnectNoQrCodeConnector,
+    injected(),
+    coinbaseConnector,
     bloctoConnector,
-    ledgerConnector,
-    trustWalletConnector,
-    binanceWeb3WalletConnector,
-    ...(cyberWalletConnector ? [cyberWalletConnector as any] : []),
   ],
 })
 
 export const CHAIN_IDS = chains.map((c) => c.id)
-
-export const isChainSupported = memoize((chainId: number) => (CHAIN_IDS as number[]).includes(chainId))
+export const isChainSupported = memoize((chainId: number | undefined) =>
+  chainId ? (CHAIN_IDS as number[]).includes(chainId) : false,
```

Comment:
I'd go with this, but this is just some code styling point of view ^^
```suggestion
  !!chainId && (CHAIN_IDS as number[]).includes(chainId),
```